features/adres.feature

```diff
@@ -49,43 +52,45 @@ Functionaliteit: Adresvelden vullen
+      | huisletter en huisnummertoevoeging begint met letter | 000000243           | St. Jacobsstraat         | St. Jacobsstraat           | 400        | L          | Toe                 |                           | St. Jacobsstraat 400 LToe    |
+      | huisletter en huisnummertoevoeging begint met cijfer | 000000255           | Cronus                   |                            | 555        | B          | 73c                 |                           | Cronus 555 B73c              |
+      | aanduiding bij huisnummer bij                        | 000000267           | Graan voor Visch         |                            | 15201      |            |                     | by                        | Graan voor Visch bij 15201   |
+      | aanduiding bij huisnummer tegenover                  | 000000279           | 1e Exloërmond            | 1e Exloërmond              | 3          |            |                     | to                        | 1e Exloërmond t/o 3          |
 
   Rule: Voor een binnenlandse locatie wordt adresregel1 gevuld met de locatiebeschrijving
 
-    Scenario: adresregel1 voor locatie       
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 000009921 |
-      En de persoon heeft de volgende 'verblijfplaats' gegevens
-      | naam                        | waarde                     |
-      | locatiebeschrijving (12.10) | Woonboot in de Grote Sloot |
+    Scenario: adresregel1 voor locatie
+    Gegeven de persoon met burgerservicenummer '000000309' heeft de volgende 'verblijfplaats' gegevens
+      | naam                  | waarde     |
+      | functie adres (10.10) | W          |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                        | waarde                            |
+      | locatiebeschrijving (12.10) | Woonboot tegenover de Grote Sloot |
+      | gemeente_code               | 0518                              |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 000009921                       |
+      | burgerservicenummer | 000000309                       |
       | fields              | adressering.adresregel1         |
       Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
       | naam        | waarde                     |
-      | adresregel1 | Woonboot in de Grote Sloot |
+      | adresregel1 | Woonboot tegenover de Grote Sloot |
 
   Rule: Voor een binnenlands adres of locatie wordt adresregel2 samengesteld conform NEN 5825:2002.
     - Veld adresregel2 wordt samengesteld uit postcode + woonplaats
```

Comment:
Uit onderstaand locatie voorbeeld blijkt dat postcode niet altijd van toepassing is. Als dat klopt dan is het wellicht wel zo duidelijk dat hier mee te nemen. Bijv. door deze regel te wijzigen in:

`Veld adresregel2 wordt samengesteld de woonplaats indien van toepassing voorafgegaan door de postcode.

Reply:
Wat jij hier beschrijft wordt in de 5e bullet toegelicht.